src/wazuh_modules/wm_sca.c

```diff
@@ -560,7 +561,7 @@ static void wm_sca_read_files(wm_sca_t * data) {
                 minfo("Starting evaluation of policy: '%s'", data->profile[i]->profile);
 
                 if (wm_sca_do_scan(profiles,vars,data,id,policy,0,cis_db_index,data->profile[i]->remote,first_scan,&checks_number) != 0) {
-                    merror("Evaluating the policy file: '%s. Set debug mode for more detailed information.", data->profile[i]->profile);
+                    merror("Evaluating the policy file: '%s'. Set debug mode for more detailed information.", data->profile[i]->profile);
```

Comment:
Let's change this error to `"Error while evaluating the policy '%s'", data->profile[i]->profile`